csharp/src/Microsoft.ML.OnnxTraining/TrainingSession.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.ML.OnnxRuntime;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.OnnxTraining
+{
+    public class TrainingSession
+    {
+        TrainingParameters m_param = new TrainingParameters();
+
+        public TrainingSession()
+        {
+        }
+
+        public TrainingParameters Parameters
+        {
+            get { return m_param; }
+        }
+
+        public void Initialize(LogLevel logLevel = LogLevel.Warning)
+        {
+            IntPtr hEnv;
+            NativeApiStatus.VerifySuccess(NativeMethods.OrtCreateEnv(logLevel, @"CSharpOnnxRuntime.Training", out hEnv));
```

Comment:
>OrtCreateEnv [](start = 56, length = 12)

Would it be possible to reuse the existing environment class and modify it as necessary?

Reply:
Yes - can you add to the OrtEnv constructor an optional parameter for the LogLevel?  That is the only reason that I went directly to the NativeMethods.